unitTests/controllers/DynamicFacetQueryControllerTest.ts

```diff
@@ -58,6 +62,10 @@ export function DynamicFacetQueryControllerTest() {
       expect(facetRequest().type).toBe(facet.facetType);
     });
 
+    it('should send the injectionDepth', () => {
+      expect(facetRequest().injectionDepth).toBe(facet.options.injectionDepth);
```

Comment:
Should we not actually set the `injectionDepth` option in the test case for it to be meaningful ?

Reply:
updated